static/src/js/util/url/mapEncoders.js

```diff
@@ -173,8 +171,7 @@ export const decodeMap = (params) => {
   if (overlaysParam) {
     decodedOverlays = {
       referenceFeatures: overlaysParam.split(',').indexOf('referenceFeatures') !== -1,
-      coastlines: overlaysParam.split(',').indexOf('coastlines') !== -1,
-      referenceLabels: overlaysParam.split(',').indexOf('referenceLabels') !== -1
```

Comment:
If we do keep this change we'll want to make sure we update https://wiki.earthdata.nasa.gov/display/EDSC/Earthdata+Search+URL+Parameters when it hits prod